.gitignore

```diff
@@ -2,3 +2,5 @@ node_modules
 jspm_packages
 .env
 dist
+config.js
+test-data.js
```

Comment:
it was never something intended to stay in the final repo, just a file i was using to see outputs from sequelize as I was writing the API docs. I think I answered my own question